src/Okta/Provider.php

```diff
@@ -175,4 +175,45 @@ public function getLogoutUrl(string $idToken, string $redirectUri = null, string
 
         return "$url?$params";
     }
+
+    /**
+     * @param string $token
+     * @param string $hint
+     *
+     * @return \Psr\Http\Message\ResponseInterface
+     */
+    public function revokeToken($token, $hint = 'access_token')
```

Comment:
can we have types please?

Reply:
(and below)

Reply:
@atymic fixed. Should probably add a check for this in the linter at some point.